Lib/glyphsLib/classes.py

```diff
@@ -3300,6 +3300,7 @@ def customValue3(self, value):
         {"plist_name": "axesValues", "object_name": "axes"},
         {"plist_name": "manualInterpolation", "converter": bool},
         {"plist_name": "properties", "type": GSFontInfoValue},
+        {"plist_name": "type", "converter": bool},
```

Comment:
is it a bool or an int? (more like an IntEnum)

Reply:
In the plist (Glyphs file) it is given as `type = variable`, or not at all. I figured interpreting the string as bool is the easiest way to interpret the value in sync with the constants INSTANCETYPESINGLE (0)/INSTANCETYPEVARIABLE (1).

I'm not familiar with IntEnum, how would the conversion be done?

Reply:
these two constants, INSTANCETYPESINGLE and INSTANCETYPEVARIABLE, _what_ are they in the Glyphs Python API? Are they integers? Strings?

if one does `type(instance.type)` what do you get?

IIUC, when serializing to plist, the the type is either omitted when == default INSTANCETYPESINGLE, or set to "variable" for INSTANCETYPEVARIABLE.

Depending on what the INSTANCETYPESINGLE/VARIABLE constants are, whether int or str, we can either use an IntEnum or a StrEnum: i.e. an Enum mixed in with an int or a str type which compares equal to those and can be used as drop-in replacement, see https://docs.python.org/3.11/library/enum.html#enum.IntEnum and https://docs.python.org/3.11/library/enum.html#enum.StrEnum) 

StrEnum was only added with 3.11 to the enum module, but one can achieve the same with 

```
from enum import Enum
class StrEnum(str, Enum):
    __str__ == str.__str__

class InstanceType(StrEnum):
    SINGLE = "single"
    VARIABLE = "variable"

assert InstanceType("variable") == InstanceType.VARIABLE == "variable"

with pytest.raises(ValueError):
    InstanceType("unknown")
```

Reply:
If the constants are integers, we can use IntEnum like so:

```python
from enum import IntEnum

class InstanceType(IntEnum):
    SINGLE = 0
    VARIABLE = 0

assert InstanceType.SINGLE == InstanceType(0) == 0
assert InstanceType.VARIABLE == InstanceType(1) == 1
with pytest.raises(ValueError):
    InstanceType(2)

assert InstanceType.VARIABLE.name.lower() == "variable"
```

Reply:
> if one does type(instance.type) what do you get?

in Glyphs: 
```
print(type(instance.type))

>> <class 'int'>
```

Reply:
I like the idea of using enums like this. I might adjust the Glyphs API in that direction.